.gitignore

```diff
@@ -26,3 +26,6 @@ tedi_test.go
 *.swp
 *.swo
 *.swn
+/toolchains/esp32s3/sdkconfig
```

Comment:
Playing a bit with the system, I think these should be generated and checked in.
Usually, I can just `rm -rf build` to get to a clean state (assuming `git` tells me nothing has changed). That's not the case if we ignore the sdkconfig files. -> We should check them in (even though that means that we are duplicating the defaults).

Reply:
I added this as part of the make clean. If the sdkconfig files are added to git, the sdkconfig.defaults files will very soon grow stale and the comparison between the config files between platforms becomes very difficult. A diff on a full sdkconfig file between esp32s3 and esp32 is 517 lines and between the defaults it is 30 lines.

Reply:
I will see if I can change the Makefiles (once committed) to conditionally set `SDKCONFIG`. If the sdkconfig file exists in the toolchain directory, write a message that a modified sdkconfig is used, otherwise set it to a path in the build-directory.

This way we can also remove the gitignore.